### ISSUE TITLE
Compiling fails using import()

### ISSUE THREAD

**Issue by pcm0nk** [JeffreyWay/laravel-mix#2020]
Im trying to run vuetify material dashboard on my vue and laravel as backend ,after modifying the vuetify package.json for laravel mix and npm ( default was for vue-cli) , running npm run dev or npm run watch commands throw this error :
```javascript
ERROR in ./resources/assets/js/router/index.js                                                                                              
Module build failed: SyntaxError: 
C:/wamp64/www/resources/assets/js/router/index.js: Unexpected token (22:28)                                                                                                                                                                              
name: name || view,                                                                                                                 
path,                                                                                                                               
component: (resovle) => import(                                                                                                                          
                        ^                                                                                                           
`@/views/${view}.vue`                                                                                                            
).then(resovle)                                                                                                                     
}  
```
its Vuetify material template that im using and tryig to compile. Here is my Webpack :
```javascript
const mix = require('laravel-mix');
const path = require('path')
mix.js('resources/assets/js/main.js', 'public/js');

mix.webpackConfig({
        output: {
        publicPath: "/",
    },
  resolve: {
    extensions: ['.js', '.vue', '.json'],
    alias: {
      'vue$': 'vue/dist/vue.esm.js',
      '@': __dirname + '/resources/assets/js'
    },
  },
})
```
My eslintrc.js
```javascript
module.exports = {
  root: true,
  env: {
    node: true
  },
  'extends': [
    'standard',
    'plugin:vue/recommended'
  ],
  rules: {
    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off'
  },
  parserOptions: {
    parser: 'babel-eslint'
  }
}
```
The Babel.config.js
```javascript
module.exports = {
  presets: [
    '@vue/app',
    "@babel/preset-stage-2"
  ]

 "plugins": ["dynamic-import-webpack"]
}
```
And packets.json
```javascript
{
  "private": true,
  "scripts": {
    "dev": "npm run development",
    "development": "node --max_old_space_size=8192 --max-old-space-size=8192 node_modules/cross-env/dist/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
    "watch": "node --max_old_space_size=8192 --max-old-space-size=8192 node_modules/cross-env/dist/bin/cross-env.js NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
    "watch-poll": "npm run watch -- --watch-poll",
    "hot": "node node_modules/cross-env/dist/bin/cross-env.js NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
    "prod": "npm run production",
    "production": "node node_modules/cross-env/dist/bin/cross-env.js NODE_ENV=production node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
  },
  "dependencies": {
    "chartist": "0.11.0",
    "node-sass": "^4.9.3",
    "sass-loader": "^7.1.0",
    "vue": "^2.5.17",
    "vue-chartist": "^2.1.2",
    "vue-meta": "^1.5.2",
    "vuetify": "^1.2.5"
  },
  "devDependencies": {
    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
    "@babel/preset-stage-2": "^7.0.0",
    "@mdi/font": "^2.5.94",
    "@vue/cli-plugin-babel": "^3.0.1",
    "@vue/cli-plugin-eslint": "^3.0.1",
    "@vue/cli-service": "^3.0.1",
    "@vue/eslint-config-standard": "^3.0.1",
    "axios": "^0.17.1",
    "babel-eslint": "^8.2.6",
    "babel-loader": "^7.1.4",
    "babel-plugin-dynamic-import-webpack": "^1.1.0",
    "babel-plugin-transform-object-rest-spread": "^6.26.0",
    "bootstrap": "^4.0.0",
    "cross-env": "^5.2.0",
    "es6-promise": "^4.2.4",
    "eslint": "^5.15.0",
    "file-loader": "^1.1.11",
    "google-maps": "3.2.1",
    "jquery": "^3.2",
    "laravel-mix": "^2.0",
    "lodash": "^4.17.4",
    "material-design-icons-iconfont": "^3.0.3",
    "popper.js": "^1.12",
    "stylus": "^0.54.5",
    "stylus-loader": "^3.0.1",
    "vue-analytics": "^5.8.0",
    "vue-i18n": "^7.4.0",
    "vue-router": "^3.0.1",
    "vue-template-compiler": "^2.5.17",
    "vuex": "^3.0.1",
    "vuex-router-sync": "^5.0.0"
  }
}
```

Im confused that if its a babel or laravel-mix issue or its rising from somewhere else that i might missed.
npm: '6.4.1',node : v10.15.1
Thanks for any help

**Comment by pcm0nk** [comment]
I added system.import instead of import in index.js and its fixed, as far as i know webpack 3 changed system.import() to import() due to its behavior is incorrect according to the spec. [why system.import is remved](https://github.com/webpack/webpack/issues/2163)
Also i noticed the  resovle typo in the code too which needs to be corrected (vuetify material dashbaord v1.0.0)

**Comment by ankurk91** [comment]
https://github.com/JeffreyWay/laravel-mix/issues/1249#issuecomment-335112415
